datarepo-clienttests/src/main/java/common/utils/BigQueryUtils.java

```diff
@@ -35,11 +37,23 @@ public static BigQuery getClientForTestUser(
         "Fetching credentials and building BigQuery client object for test user: {}",
         testUser.name);
 
+    RetrySettings retrySettings =
+        RetrySettings.newBuilder()
+            .setInitialRetryDelay(Duration.ofMillis(1000L))
+            .setMaxRetryDelay(Duration.ofMillis(32_000L))
```

Comment:
FWIW, I think you can do stuff like `Duration.ofMillis(TimeUnit.SECONDS.toMillis(32))` or `Duration.of(32, TimeUnit.SECONDS.toChronoUnit())` which is maybe a touch easier to read

Reply:
I think `Duration.ofSeconds()` works well here.
```suggestion
            .setInitialRetryDelay(Duration.ofSeconds(1))
            .setMaxRetryDelay(Duration.ofSeconds(32))
```

Reply:
Good point!